### DISCUSSION TITLE
V2.9.13  form.render(); 更新表单，其中下拉选项，显示异常，请问你们也会这样吗？

### DISCUSSION THREAD

**Discussion by tammyqqcom** [layui/layui#2069] [问答]
layui.layer.open({
            title : title,
            type : 2,
            offset:'r',
            area : ["80%","100%"],
            content : "info.html",
            success : function(layero, index){
                var body = layui.layer.getChildFrame('body', index);
                if(edit){
                    body.find("#getCodeDiv").remove();
                    body.find("#id").val(edit.id);                   
                    body.find("#inGuaranteeSelect").val(edit.inGuarantee);                    
                    form.render();
                }
                setTimeout(function(){
                    layui.layer.tips('点击此处返回列表', '.layui-layer-setwin .layui-layer-close', {
                        tips: 3
                    });
                },500)
            }
        })

<body class="childrenBody">
<form class="layui-form linksAdd" lay-filter="demo-filter">
    <input type="hidden" value="" id="id" name="id">
    <div class="layui-form-item">        
            <label class="layui-form-label required">质保范围内</label>
            <div class="layui-input-block">
                <select name="inGuarantee" id="inGuaranteeSelect" >
                    <option value="1">是</option>
                    <option value="0">否</option>
                </select>
            </div>         
    </div>
</form>
</body>

**Comment by LSL1618** [comment]
好歹截图发出来吧！
